.pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-step-template.yaml

```diff
@@ -64,6 +64,56 @@ steps:
       sudo go test -count=1 datapath_linux_test.go -timeout 1m -tags connection -run ^TestDatapathLinux$ -tags=connection,integration -isDualStack=true
       echo "Dualstack Overlay Linux datapath IPv4 test"
       sudo go test -count=1 datapath_linux_test.go -timeout 1m -tags connection -run ^TestDatapathLinux$ -tags=connection,integration
+      echo "cleaning up load-test namespace"
+      kubectl delete ns load-test
     retryCountOnTaskFailure: 3
     name: "DualStack_Overlay_Linux_Tests"
     displayName: "DualStack Overlay Linux Tests"
+
+  - task: AzureCLI@2
+    inputs:
+      azureSubscription: $(AZURE_TEST_AGENT_SERVICE_CONNECTION)
+      scriptLocation: "inlineScript"
+      scriptType: "bash"
+      addSpnToEnvironment: true
+      inlineScript: |
+        make -C ./hack/aks windows-nodepool-up AZCLI=az SUB=$(SUB_AZURE_NETWORK_AGENT_TEST) CLUSTER=${{ parameters.clusterName }}
+        echo "Windows node are successfully added to Dualstack Overlay Cluster"
+        kubectl cluster-info
+        kubectl get node
+        kubectl get po -owide -A
+        echo "Deploying windows CNS"
+        kubectl apply -f test/integration/manifests/cns/daemonset-windows.yaml
+        echo "wait for CNS pods deployment ready"
+        kubectl rollout status deployment azure-cns-win --timeout=30s
+        kubectl get po -owide -A
+        echo "Upload CNI"
+        dropgz_version=$(make cni-dropgz-version-test)
+        export DROP_GZ_URL=$( make cni-dropgz-test-image-name-and-tag OS='windows' ARCH='amd64' OS_VERSION='ltsc2022' CNI_DROPGZ_VERSION=${dropgz_version})
+        envsubst < ./test/integration/manifests/cni/cni-installer-v1-windows.yaml | kubectl apply -f -
+    name: "Add_Windows_Node"
+    displayName: "Add windows node on DualStackOverlay cluster"
+
+  - script: |
+      cd test/integration/load
+      echo "DualStack Overlay Windows control plane CNS validation test"
+      sudo go test -timeout 30m -tags load -run ^TestLoad$ -tags=load -os=windows
+      echo "restart kubeproxy for windows pod internet connectivity"
+      priviledgedPodList=`kubectl get pods -n kube-system | grep privileged | awk '{print $1}'`
+      for priviledgedPod in $priviledgedPodList; do
+          kubectl -n kube-system exec priviledgedPod -- powershell Restart-service kubeproxy
+          kubectl -n kube-system exec priviledgedPod -- powershell Get-service kubeproxy
+      done
```

Comment:
let's use golang for this test as well

Reply:
I think it's better to enable kubeproxy here because this is not a test case.